docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -195,18 +197,18 @@ where `keycloak.version` should be set to `23.0.0` or higher.
 You should be able to access your Keycloak Server at http://localhost:8180[localhost:8180].
 
 To access the Keycloak Administration Console, log in as the `admin` user.
-Username should be `admin` and password `admin`.
+The username is `admin` and the password is `admin`.
 
-Import the link:{quickstarts-tree-url}/security-openid-connect-web-authentication-quickstart/config/quarkus-realm.json[realm configuration file] to create a new realm.
+To create a new realm, import the link:{quickstarts-tree-url}/security-openid-connect-web-authentication-quickstart/config/quarkus-realm.json[realm configuration file].
 For more information, see the Keycloak documentation about how to https://www.keycloak.org/docs/latest/server_admin/index.html#configuring-realms[create and configure a new realm].
 
 == Run the application in dev and JVM modes
 
-To run the application in a dev mode, use:
+To run the application in dev mode, use:
 
 include::{includes}/devtools/dev.adoc[]
 
-When you're done playing with dev mode, you can run it as a standard Java application.
+When you are finished exploring the application in dev mode, you can run it as a standard Java application.
```

Comment:
```suggestion
After exploring the application in dev mode, you can run it as a standard Java application.
```

Reply:
Reads much more smoothly, thanks for the suggestion!